WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -70,51 +70,18 @@ private extension ItemListView {
     }
```

Comment:
It's unrelated to this PR, but for some reason we hide `bannerCardView` for `dynamicTypeSize.isAccessibilitySize`. I cannot find a good argument for it. 

We can leave something like this if we don't want for it to take too much space:

```swift
            bannerCardView
                .padding(.horizontal, Constants.bannerCardPadding)
                .dynamicTypeSize(...DynamicTypeSize.accessibility1)
```

Reply:
Interestingly, I'm still seeing the notice/banner in the largest font size. Capping the dynamic type size sounds good, added in https://github.com/woocommerce/woocommerce-ios/pull/15197/commits/8c96d97a8103c203be47281652fad6215317bd8e.

small font size | largest font size
-- | --
![Simulator Screenshot - iPad Pro 11-inch (M4) - 2025-02-19 at 17 59 23](https://github.com/user-attachments/assets/c5fe7cfe-6280-4778-a22d-f04fdd01953b) | ![Simulator Screenshot - iPad Pro 11-inch (M4) - 2025-02-19 at 17 59 42](https://github.com/user-attachments/assets/adb09e23-1894-4ba5-94bc-1b5eb28c271d)

Reply:
Interesting. For me, it disappears for accessibility size 🤔 Just like this check supposed to do: `if !dynamicTypeSize.isAccessibilitySize`

Reply:
It's the toggle in the "larger text" setting, and when the "larger" font size is selected (from the documentation it sounds pretty vague) right?

<img src="https://github.com/user-attachments/assets/ff968f87-a765-4f74-919f-d54734f4919b" width="300" />

In any case, I'd keep the visibility logic as it is so as not to risk any regressions. My guess is that we might have decided to hide it in larger font size so that the whole item list is not covered by the notice.